Taskfile.yml

```diff
@@ -127,7 +127,19 @@ tasks:
       - rm -rf output
       - mkdir output
       - podman pull ${REGISTRY}/${PROJECT}/${IMAGE}:${TAG}
-      - podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t -v ./config.toml:/config.toml:ro -v ./output:/output -v /var/lib/containers/storage:/var/lib/containers/storage quay.io/centos-bootc/bootc-image-builder:latest --type raw --rootfs btrfs --local ${REGISTRY}/${PROJECT}/${IMAGE}:${TAG}
+      - podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t -v ./config.toml:/config.toml:ro -v ./output:/output -v /var/lib/containers/storage:/var/lib/containers/storage registry.playtron.one/tools/bootc-image-builder:latest --type raw --rootfs btrfs ${REGISTRY}/${PROJECT}/${IMAGE}:${TAG}
+
+  disk-image:bootc:local:
+    desc: Build the OS image from a local container using bootc image builder.
+    preconditions:
+      - sh: test "$EUID" == 0
+        msg: "This task must be run as root"
+      - sh: test -n "$IMAGE"
+        msg: "A value for IMAGE must be provided"
+    cmds:
+      - rm -rf output
+      - mkdir output
+      - podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t -v ./config.toml:/config.toml:ro -v ./output:/output -v /var/lib/containers/storage:/var/lib/containers/storage registry.playtron.one/tools/bootc-image-builder:latest --type raw --rootfs btrfs localhost/${IMAGE}:${TAG}
```

Comment:
Two minor things:

(1) I tried using the full registry instead of `localhost/${IMAGE}:${TAG}` but got this error below.  We should probably do a `podman pull` first.

```
bootc-image-builder no longer pulls images, make sure to pull it before running bootc-image-builder:
```

(2) Should we host the forked BIB image on GHCR at `ghcr.io/playtron-os/bootc-image-builder:latest`?

Reply:
(1) That is strange, we are already pulling the image in the previous step.
(2) That is a good idea.